### PR TITLE
Add Fluent CSS and Controls library

### PR DESCRIPTION
**Pull request by vishwam**
1. Added Azure IoT Fluent CSS and Controls library
2. Ran into compile errors with sass, so updated to latest version of node-sass, react-scripts and a few other packages to more closely align with baseline.
3. Turns out the JS files require the css file directly (instead of the scss), so had to update node-sass-chokidar and add a new importer to map ~ to node_modules. We should eventually remove the direct css import because react-scripts has built-in support for sass, but that requires touching >100 files so leaving that for a different PR.
4. Updated _themes.scss to import and extend the CSS library's colors, so we can start using its @themify mixin and color variables everywhere.
5. Updated the RM Shell component to use the Fluent Shell (which specifies a common way to support themes and RTL) as an example, and change some of its styling to use the fluent colors.

### CODE REVIEW

File: src/components/shared/indicator/indicator.scss [hunk 1]
```diff
@@ -242,7 +242,7 @@
       width: 100%;
       text-align: left;
       opacity: 0;
-      transform: translateX(calc(0 '-' 3px));
+      transform: translateX(calc("0 - 3px"));
```
Comment (Coord) [inline src/components/shared/indicator/indicator.scss:245]: nit: is this the same as -3px

File: src/styles/_themes.scss [hunk 2]
```diff
@@ -1,383 +1,389 @@
...
+  // Modal - START
+  colorModalText: #333,
+  colorModalBackground: $colorWhite,
+  colorModalBorder: #136BFB,
+  colorModalDropShadow: rgba(0, 0, 0, 0.1),
+  // Modal -  END
+);
+
+// as a final step, import the default colors (this will preserve any overrides specified above):
+@import "~@microsoft/azure-iot-ux-fluent-css/src/colors";
```
Comment (Coord) [inline src/styles/_themes.scss:385]: nit: put this import in _colors.scss, so this file can just be the overrides

**Review by Coord — APPROVED** [review]

**Review by vincenzocaruso — APPROVED** [review]

File: src/styles/_themes.scss (hunk 2 above)
Comment (vishwam) [reply src/styles/_themes.scss:385]: Nope, can't do this. A lot of their existing css files expect $themes to exist in _themes.scss, so we have to bring in the CSS colors here. We can remove this file in the future when we're standardizing on baseline

File: src/components/shared/indicator/indicator.scss (hunk 1 above)
Comment (vishwam) [reply src/components/shared/indicator/indicator.scss:245]: I think so, but I'm sure there was a reason why it had to be specified like this, so I'm leaving it as-is (I had to wrap it in quotes to pass the build). We'll have to replace this with the fluent progressbar in a different PR anyway.

**Pull request merged**